iis-3rdparty-pig-avrostorage/src/test/java/org/apache/pig/piggybank/test/storage/avro/TestAvroStorage.java

```diff
@@ -16,8 +16,18 @@
  */
 package org.apache.pig.piggybank.test.storage.avro;
 
+import static junit.framework.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
```

Comment:
Change assertEquals import to org.junit.Assert.assertEquals. I believe that asserts from junit.framework.Assert are deprecated.